### ISSUE TITLE
Linter is not being triggered on vue files

### ISSUE THREAD

**Issue by ellipticaldoor** [SublimeLinter/SublimeLinter-eslint#207]
So I install sublimelinter, sublimelinter-eslint and vue syntax highlight but it looks like the linter is working on .js files but not on .vue

There is something special that I have to do to trigger the lint on .vue files?

**Comment by braver** [comment]
Master is ahead of the release here, I think we already fixed it but I need to test some cases to verify. If you want, you can manually install and test if that fixes the problem for you.

**Comment by kaste** [comment]
selector must include text.html.vue. can't type that well on mobile.

**Comment by ellipticaldoor** [comment]
Yes! It works :) Thank you

**Comment by braver** [comment]
🎉

**Comment by ellipticaldoor** [comment]
Sorry, it stopped working and now I'm getting this error

``` python
Traceback (most recent call last):
  File "/Users/miguel/Library/Application Support/Sublime Text 3/Packages/SublimeLinter/lint/backend.py", line 97, in execute_lint_task
    errors = linter.lint(code, view_has_changed, settings) or []
  File "/Users/miguel/Library/Application Support/Sublime Text 3/Packages/SublimeLinter/lint/linter.py", line 813, in lint
    for m in self.find_errors(output):
  File "/Users/miguel/Library/Application Support/Sublime Text 3/Packages/SublimeLinter-eslint/linter.py", line 47, in find_errors
    content = json.loads(output)
  File "./python3.3/json/__init__.py", line 316, in loads
  File "./python3.3/json/decoder.py", line 351, in decode
  File "./python3.3/json/decoder.py", line 369, in raw_decode
ValueError: No JSON object could be decoded
```

**Comment by ellipticaldoor** [comment]
If I close the error and save the linter works again but not on vue files.

I don't know what I did before but now linter on vue files is not working for me.

@kaste what do you mean about "selector must include text.html.vue" ?

**Comment by ellipticaldoor** [comment]
My bad, I also need to install vue-syntax-highlight from master
